examples/mysql/mysql_status.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
```

Comment:
how would you recommend running this same script multiple times against multiple different mysql servers?